File: src/java/Application.java
package com.sample;

import java.sql.*;
import java.io.*;

public class Application {
    public static void main(String[] args) throws Exception {
        System.out.println("Sample Java Application");
        
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        String userInput = reader.readLine();
        
        // Potential SQL injection
        Connection conn = DriverManager.getConnection("jdbc:mysql://localhost/db");
        Statement stmt = conn.createStatement();
        ResultSet rs = stmt.executeQuery("SELECT * FROM users WHERE name = '" + userInput + "'");
        
        // Potential command injection
        Runtime.getRuntime().exec("cmd /c " + userInput);
        
        // Potential path traversal
        FileInputStream fis = new FileInputStream("/data/" + userInput);
    }
}

File: src/kotlin/Application.kt
package com.sample

import java.io.File
import java.sql.DriverManager

fun main(args: Array<String>) {
    println("Sample Kotlin Application")
    
    val userInput = readLine() ?: ""
    
    // Potential SQL injection
    val conn = DriverManager.getConnection("jdbc:mysql://localhost/db")
    val stmt = conn.createStatement()
    stmt.executeQuery("SELECT * FROM users WHERE name = '$userInput'")
    
    // Potential command injection
    Runtime.getRuntime().exec("cmd /c $userInput")
    
    // Potential path traversal
    val file = File("/data/$userInput")
    println(file.readText())
}
